tracing-core/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 0.1.32 (October 12, 2023)
+
+### Fixed
+
+- Fix typo in `field` docs (#2611)
+- Remove duplicate wording (#2674)
+- Remove usage of 0.2 terminology (#2728)
```

Comment:
I wouldn't include this in the changelog, since it fixes an issue that was only introduced while doing backports --- it doesn't fix something that was actually released.